scripts/packaging/mac_distro.py

```diff
@@ -159,19 +186,31 @@ def create_webots_bundle(self):
 
         print('Done.\n')
 
-    def make_dir(self, directory):
+    def make_dir(self, directory=''):
```

Comment:
Why did you add a default empty value for the directory?
If directory is empty, then this function does nothing and should probably not be called.

Reply:
I used it before to create the 'Contents' folder, but I don't use it any more, so I just removed it.